bulk/management/commands/data_quality.py

```diff
@@ -0,0 +1,184 @@
+import base62
+from django.core.management.base import BaseCommand
+from django.db.models import F, Count, Avg
+from openstates_metadata import STATES_BY_NAME
+from openstates.data.models import (
+    LegislativeSession,
+    Bill,
+    BillAbstract,
+    BillAction,
+    BillTitle,
+    BillIdentifier,
+    RelatedBill,
+    BillSponsorship,
+    BillDocument,
+    BillVersion,
+    BillDocumentLink,
+    BillVersionLink,
+    BillSource,
+    VoteEvent,
+    PersonVote,
+    VoteCount,
+    VoteSource,
+)
+from utils.common import abbr_to_jid
+from utils.orgs import get_chambers_from_abbr
+from collections import defaultdict
+from statistics import mean
+
+# Loads the global bill array with all bills from given state and session to use
+#   when creating the json
+def load_bills(state, session):
+    sobj = LegislativeSession.objects.get(
+        jurisdiction_id=abbr_to_jid(state), identifier=session
+    )
+    bills = Bill.objects.filter(legislative_session=sobj).prefetch_related("actions", "sponsorships", "votes", "votes__counts", "sources")
+    return bills
+
+
+def get_available_sessions(state):
+    return sorted(
+        s.identifier
+        for s in LegislativeSession.objects.filter(jurisdiction_id=abbr_to_jid(state))
+    )
+
+def total_bills_per_session(bills, chambers):
+    total_bills_per_session = defaultdict(list)
+    for chamber in chambers:
+        chamber_name = chamber.name.lower()
+        total_bills = bills.filter(from_organization=chamber).count()
+        # Set variables to empty strings in case any info is blank
+        latest_bill_created_id = ""
+        latest_bill_created_date = ""
+        bill_with_latest_action_id = ""
+        latest_action_date = ""
+        latest_action_description = ""
+
+        if total_bills > 0:
+            latest_bill = bills.filter(from_organization=chamber).latest("created_at")
+            latest_bill_created_id = latest_bill.identifier
+            latest_bill_created_date = latest_bill.created_at.strftime("%Y-%m-%d")
+            bill_with_latest_action = bills.filter(from_organization=chamber).latest("actions__date")
+            # In case bills don't have actions
+            if bill_with_latest_action.actions.count() > 0:
+                bill_with_latest_action_id = bill_with_latest_action.identifier
+                latest_action = bill_with_latest_action.actions.latest("date")
+                latest_action_date = latest_action.date
+                latest_action_description = latest_action.description
+
+        total_bills_per_session[chamber_name].append({
+            "chamber": chamber_name,
+            "total_bills": total_bills,
+            "latest_bill_created_id": latest_bill_created_id,
+            "latest_bill_created_date": latest_bill_created_date,
+            "bill_id_with_latest_action": bill_with_latest_action_id,
+            "latest_action_date": latest_action_date,
+            "latest_action_description": latest_action_description}
```

Comment:
thinking this might be hard to display in any meaningful way, maybe we'll drop this one